improbable/bazelrc

```diff
@@ -0,0 +1,14 @@
+common:imp-ci --curses=no
```

Comment:
I see three cases where we're going to run a build here:

1. Locally, during iteration
	- This will not use any of the options from this file.
1. CI, premerge
	- This will use the `imp-ci` config, as set in `improbable/run_dockerized_tests.sh`
1. Release and push
	- This will use the `imp-release` config, which includes `imp-ci` plus hardwiring the linux platform for binaries.